<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="25789" started-at="2013-02-05T10:17:24Z" finished-at="2013-02-05T10:17:50Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="25789" started-at="2013-02-05T10:17:24Z" finished-at="2013-02-05T10:17:50Z">
      <class name="com.YouMagicPro.LKA_tests.DiscSpaceTest">
        <test-method status="PASS" signature="startWebDriver()[pri:0, instance:com.YouMagicPro.LKA_tests.DiscSpaceTest@3a64b5ab]" name="startWebDriver" is-config="true" duration-ms="1795" started-at="2013-02-05T14:17:24Z" finished-at="2013-02-05T14:17:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startWebDriver -->
        <test-method status="FAIL" signature="discSpaceTest()[pri:0, instance:com.YouMagicPro.LKA_tests.DiscSpaceTest@3a64b5ab]" name="discSpaceTest" duration-ms="15441" started-at="2013-02-05T14:17:26Z" finished-at="2013-02-05T14:17:41Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [199.06] but found [199.06 МБ]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [199.06] but found [199.06 МБ]
	at org.testng.Assert.fail(Assert.java:89)
	at org.testng.Assert.failNotEquals(Assert.java:489)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:160)
	at com.YouMagicPro.LKA_tests.TestBase.assertEquals(TestBase.java:51)
	at com.YouMagicPro.LKA_tests.DiscSpaceTest.discSpaceTest(DiscSpaceTest.java:16)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1197)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1122)
	at org.testng.TestNG.run(TestNG.java:1030)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- discSpaceTest -->
        <test-method status="PASS" signature="closeSession()[pri:0, instance:com.YouMagicPro.LKA_tests.DiscSpaceTest@3a64b5ab]" name="closeSession" is-config="true" duration-ms="8538" started-at="2013-02-05T14:17:41Z" finished-at="2013-02-05T14:17:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeSession -->
      </class> <!-- com.YouMagicPro.LKA_tests.DiscSpaceTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
